test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/math/BigIntegerConstructorsTest.scala

```diff
@@ -121,7 +121,7 @@ class BigIntegerConstructorsTest {
   }
 
   @Test def testConstructorBytesZero(): Unit = {
-    val aBytes = Array[Byte](0, 0, 0, -0, +0, 0, -0)
+    val aBytes = Array[Byte](0, 0, 0, -0, 0, 0, -0)
```

Comment:
Are we going to add these back when M4 comes?

Reply:
Probably not. They were not consistently used in here. It seems like they were `+0` instead of `0` by "chance".

We could add a dedicated test somewhere else, but that would be redundant with [the partest](https://github.com/scala/scala/pull/6416/files#diff-2ca2635e3d3c286af0041a1b6e77bd0a) anyway.